docs/markdown/Fs-module.md

```diff
@@ -34,6 +40,23 @@ by the string is a symbolic link.
 
 ## File Parameters
 
+### is_absolute
+
+Return a boolean indicating if the path string specified is absolute, WITHOUT expanding `~`.
```

Comment:
Missing "Since 0.54.0" because the fs module has been released in 0.53.0 already.